modules/core/src/main/java/org/apache/ignite/internal/binary/streams/BinaryMemoryAllocator.java

```diff
@@ -19,22 +19,17 @@
 
 import java.util.ArrayDeque;
 import java.util.Arrays;
-import org.apache.ignite.IgniteSystemProperties;
 import org.apache.ignite.internal.util.typedef.internal.U;
 
 import static org.apache.ignite.IgniteSystemProperties.IGNITE_MARSHAL_BUFFERS_PER_THREAD_POOL_SIZE;
 import static org.apache.ignite.IgniteSystemProperties.IGNITE_MARSHAL_BUFFERS_RECHECK;
+import static org.apache.ignite.internal.binary.streams.BinaryMemoryAllocatorChunk.DFLT_MARSHAL_BUFFERS_PER_THREAD_POOL_SIZE;
+import static org.apache.ignite.internal.binary.streams.BinaryMemoryAllocatorChunk.DFLT_MARSHAL_BUFFERS_RECHECK;
 
 /**
  * On-heap memory allocator.
  */
-public abstract class BinaryMemoryAllocator {
-    /** @see IgniteSystemProperties#IGNITE_MARSHAL_BUFFERS_RECHECK */
-    public static final int DFLT_MARSHAL_BUFFERS_RECHECK = 10000;
```

Comment:
What is a reason to move this fields? Variables moved to another class that is still within internal package. Also THREAD_LOCAL variable in this class is still public and used in BinaryStreams.